app/controllers/concerns/authorizable.rb

```diff
@@ -3,9 +3,19 @@ def authorize_user
     return unless current_user.phone_enabled?
 
     if user_fully_authenticated?
-      redirect_to account_url
+      add_methods_else_account_url
     elsif current_user.two_factor_enabled?
       redirect_to user_two_factor_authentication_url
     end
   end
+
+  private
+
+  def add_methods_else_account_url
+    if current_user.all_two_factors_enabled?
```

Comment:
I'm uncomfortable with `all_two_factors_enabled?` rather than `enough_two_factors_enabled?` because we really just want two factors rather than all three (for those who have piv/cac available). Not critical since I'll probably do a round of policy object building in a sprint or two.

Reply:
I thought about it and originally coded it to have just two.  However, what happens when we introduce a new two factor option?  They will never see it if we don't present it to them with this screen.